CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing to OSRD
+
+## Where to start ?
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+
+If you are brand new to OSRD or open-source development, we recommend going through the [GitHub issues](https://github.com/DGEXSolutions/osrd/issues) tab to find issues that interest you. There are number of issues listed under `good first issue` where you could start out.
+
+## Contribute code
+
+Commit style are detailed in the [osrd.fr/docs](https://osrd.fr/en/docs/guides/contribute/code/).
+
+### Pull request / code review
+
+- Every PR, good or bad, is an act of generosity.
+  Opening with a positive comment will help the author feel rewarded, and your subsequent remarks may be heard more clearly.
+  You may feel good also.
+- Add a description to PRs to explain what they do and why.
+- Each PR must be linked to at least one github issue attached to the PI board to simplify tracking of the work done in a sprint.
```

Comment:
The PI board is private, as well as the agile stuff. This document should be written with external contributors in mind.

If we want to document this it should be in another document, specifically for SNCF employees. Either on the website, or as an internal document if we want to keep it somehow private.